src/Raven.Server/Extensions/LoggerExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Raven.Server
+{
+    public static class LoggerExtensions
```

Comment:
Remove this class.
You are forcing us to evaluate the log string even if logging is disabled, which is something that we don't want.

See the reasoning here: https://ayende.com/blog/174530/designing-ravendbs-debug-log-system